File: java-script/Countdown.java
// 
import java.util.Scanner;

public class Countdown {
    public static void main(String[] args) throws InterruptedException {
        Scanner input = new Scanner(System.in);
        System.out.print("Masukkan waktu (detik): ");
        int detik = input.nextInt();

        while (detik > 0) {
            System.out.print("Waktu tersisa: " + detik + " detik\r");
            Thread.sleep(1000);
            detik--;
        }
        System.out.println("\nWaktu habis!");
        input.close();
    }
}

File: python-script/countdown.py

import time

def countdown(seconds):
    while seconds > 0:
        print(f"Waktu tersisa: {seconds} detik", end='\r')
        time.sleep(1)
        seconds -= 1
    print("\nWaktu habis!")

if __name__ == "__main__":
    sec = int(input("Masukkan waktu (detik): "))
    countdown(sec)
